cdap-examples/SpamClassifier/src/main/scala/co/cask/cdap/examples/sparkstreaming/SpamClassifierProgram.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.examples.sparkstreaming
+
+import co.cask.cdap.api.common.Bytes
+import co.cask.cdap.api.spark.{SparkExecutionContext, SparkMain}
+import com.google.common.base.Strings
+import kafka.serializer.{DefaultDecoder, StringDecoder}
+import org.apache.spark.SparkContext
+import org.apache.spark.mllib.classification.{NaiveBayes, NaiveBayesModel}
+import org.apache.spark.mllib.feature.HashingTF
+import org.apache.spark.mllib.linalg.Vector
+import org.apache.spark.mllib.regression.LabeledPoint
+import org.apache.spark.rdd.RDD
+import org.apache.spark.streaming.kafka.KafkaUtils
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import org.slf4j.{Logger, LoggerFactory}
+
+/**
+  * Consumes messages from kafka topics and classify them as spam or ham based on NaiveBayes model.
+  * A comma separated list of brokers and topics must be provided as runtime arguments to this program.
+  */
+class SpamClassifierProgram extends SparkMain {
+
+  private final val LOG: Logger = LoggerFactory.getLogger(classOf[SpamClassifierProgram])
```

Comment:
Define logger in companion object to avoid any possible failure in serializing logback logger.